src/main/java/com/socrata/api/HttpLowLevel.java

```diff
@@ -78,6 +81,8 @@
 
     public static final GenericType<List<Object>> MAP_OBJECT_TYPE = new GenericType<List<Object>>() {};
 
+    private final ObjectMapper mapper;
```

Comment:
Minor suggestion: since ObjectMapper is stateless, you can maintain a single ObjectMapper object as singleton across the code base.